hs_core/templates/pages/genericresource.html

```diff
@@ -830,8 +831,22 @@ <h3>Resource Specific</h3>
       const CURRENT_USER_ID = {{ request.user.id }};
       const SHORT_ID = "{{ cm.short_id }}";
       const STATIC_URL = "{{ STATIC_URL }}";
-      let spatial_coverage_type = "{{ spatial_coverage.type }}";
+      {% if show_manage_access %}
```

Comment:
I don't recall coming across something like this before, with the Django template language, in the middle of a frontend <script> tag. Also, should this specify text/javascript or be more descriptive than just <script>?

Reply:
I've seen it in this codebase before, though I don't have an opinion yet whether it's a good thing or not.  I don't have any ideas for alternative implementations of this.

Reply:
Looking into this more, I think it make sense to split these into some ajax calls when vue needs them instead of on the html.

Reply:
@sblack-usu - The advantage of having it this way is that Vue gets the data as soon as the template is passed to the client. Having ajax calls to get the data for each Vue instance would add unnecessary server load. 

@cuahsimarko - Django template language works in the entire .html file, even inside inline `script` tags. However, it would not work inside .js files which I think is what you mean.

Reply:
Hopefully, we can aim to have less entangled code and more clear handoffs between contexts Django template/HTML/CSS/DOM .... JavaScript/Vue templates/components .... Python/backend/API